dataframe-jupyter/build.gradle.kts

```diff
@@ -65,7 +65,7 @@ tasks.processJupyterApiResources {
 }
 
 tasks.test {
-    maxHeapSize = "2048m"
+    maxHeapSize = "8192m"
```

Comment:
don't forget to make all these changes in master as well!

Usually these release branches just contain the version change commit (and maybe the -n patch) but nothing else